opentdf-example.yaml

```diff
@@ -66,7 +66,31 @@ server:
       #
       #  [matchers]
       #  m = g(r.sub, p.sub) && globOrRegexMatch(r.res, p.res) && globOrRegexMatch(r.act, p.act) && globOrRegexMatch(r.obj, p.obj)
-
+  cors:
+    enabled: false
+    # '*' to allow any origin or a specific domain like 'https://yourdomain.com'
+    allowedorigins: '*'
```

Comment:
@pflynn-virtru having a wildcard in the example is a footgun IMO. I would advise using `localhost:*` if we wanted any kind of default value that would be convenient for developers.